packages/jmix-react-ui/rollup.config.js

```diff
@@ -48,7 +48,8 @@ export default {
     commonjs({
       namedExports: {
         'invariant': ['default'],
-      }
+        'node_modules/react-is/index.js': ['isFragment', 'isMemo']
```

Comment:
I think we should put `react-is` under `external` instead.

Is this related to tests or sorting of generators?